src/fontra/views/editor/panel-selection-transformation.js

```diff
@@ -0,0 +1,542 @@
+        this.rotation = value;
+        return value;
+      },
+    });
+
+    let button_skew = html.createDomElement("icon-button", {
+      src: "/tabler-icons/angle.svg",
+      onclick: (event) =>
+        this._transformLayerGlyph({
+          skewX: this.skewX,
+          skewY: this.skewY,
+          undoName: "slant",
+        }),
+      class: "ui-form-icon ui-form-icon-button",
+      /*       "data-tooltip": "Slant",
+      "data-tooltipposition": "left", */
+    });
+
+    formContents.push({
+      type: "edit-number-x-y",
+      key: '["selectionTransformationSkew"]',
+      label: button_skew,
+      fieldX: {
+        key: '["selectionTransformationSkewX"]',
+        id: "selection-transformation-skewX",
+        value: this.skewX,
+        getValue: (layerGlyph, layerGlyphController, fieldItem) => {
+          return fieldItem.value;
+        },
+        setValue: (layerGlyph, layerGlyphController, fieldItem, value) => {
+          this.skewX = value;
+          return value;
+        },
+      },
+      fieldY: {
+        key: '["selectionTransformationSkewY"]',
+        id: "selection-transformation-skewY",
+        value: this.skewY,
+        getValue: (layerGlyph, layerGlyphController, fieldItem) => {
+          return fieldItem.value;
+        },
+        setValue: (layerGlyph, layerGlyphController, fieldItem, value) => {
+          this.skewY = value;
+          return value;
+        },
+      },
+    });
+
+    formContents.push({ type: "divider" });
+
+    formContents.push({
+      type: "icons",
+      label: "Flip",
+      auxiliaryElements: [
+        html.createDomElement("icon-button", {
+          src: "/tabler-icons/flip-vertical.svg",
+          onclick: (event) =>
+            this._transformLayerGlyph({
+              scaleX: -1,
+              scaleY: 1,
+              undoName: "flip vertically",
+            }),
+          /* "data-tooltip": "Flip vertically",
+          "data-tooltipposition": "left", */
+        }),
+        html.createDomElement("icon-button", {
+          src: "/tabler-icons/flip-horizontal.svg",
+          onclick: (event) =>
+            this._transformLayerGlyph({
+              scaleX: 1,
+              scaleY: -1,
+              undoName: "flip horizontally",
+            }),
+          /* "data-tooltip": "Flip horizontally",
+          "data-tooltipposition": "left", */
+        }),
+      ],
+    });
+
+    this._formFieldsByKey = {};
+    for (const field of formContents) {
+      if (field.fieldX) {
+        this._formFieldsByKey[field.fieldX.key] = field.fieldX;
+        this._formFieldsByKey[field.fieldY.key] = field.fieldY;
+      } else {
+        this._formFieldsByKey[field.key] = field;
+      }
+    }
+
+    this.infoForm.setFieldDescriptions(formContents);
+    if (glyphController) {
+      await this._setupSelectionInfoHandlers(glyphName);
+    }
+  }
+
+  _getSelectedBounds(layerGlyph, pointIndices, componentIndices) {
```

Comment:
@justvanrossum This is the context related to 'bounds of a layerGlyph component'